variables.tf

```
@@ -41,67 +47,14 @@ variable "prevent_destroy" {
   default     = true
 }
 
-variable "purpose" {
-  type        = string
-  description = "The immutable purpose of the CryptoKey. Possible values are ENCRYPT_DECRYPT, ASYMMETRIC_SIGN, and ASYMMETRIC_DECRYPT."
-  default     = "ENCRYPT_DECRYPT"
-}
-
-variable "set_owners_for" {
-  description = "Name of keys for which owners will be set."
-  type        = list(string)
+variable "acl" {
+  description = "Access control list for the managed keys."
+  type        = list(any)
   default     = []
 }
 
-variable "owners" {
-  description = "List of comma-separated owners for each key declared in set_owners_for."
-  type        = list(string)
+variable "key_opts" {
+  description = "Specifies key specific options."
+  type        = any
```

IIUC this is a list?
```suggestion
  type        = list(any)
```